tox.ini

```diff
@@ -23,6 +24,9 @@ commands =
 whitelist_externals =
     make
 
+# needed for coverage to work
+usedevelop=True
```

Comment:
Could you explain this a bit better to me?

Reply:
`--cov=cocotb` matches against `./cocotb`. If we are not using an `-e` install (which this forces), then no coverage is collected for those files.

This seems to be how I've seen the problem handled elsewhere.